pkg/scraper/aws/rds.go

```diff
@@ -63,7 +63,8 @@ func extractRds(ctx context.Context, awsClient interfaces.AWSClient) (*types.Inv
 		rdsClusters = append(rdsClusters, types.RDSCluster{
 			Engine: *instance.Engine,
 			VersionedResource: types.VersionedResource{
-				Name:           *instance.DBClusterIdentifier,
+				ID:             *instance.DBClusterIdentifier,
+				Kind:           types.KindRDSCluster,
 				Parents:        []types.ParentResource{{Kind: types.KindAWSAccount, ID: awsClient.GetAccountId()}},
 				Arn:            *instance.DBClusterArn,
 				Version:        *instance.EngineVersion,
```

Comment:
I know this is an artifact of the AWS SDK, but we should probably check that these values aren't nil so we don't have to worry about random nil pointer exceptions popping up over time. I have a helpful function I have used in the past that can help with this work:

https://github.com/chanzuckerberg/sastisfaction/blob/0b43f0c90bb958c5f205930abcffa2d9c4ddc685/pkg/ghapp/gh_context.go#L112

Reply:
then you can say something like

~~~go
if anyNil(blah1, blah2, blah3, blah4) {
  return errors.New("can't be nil!")
}
~~~